shared/httpHandler/requestTimeout.js

```diff
@@ -0,0 +1,13 @@
+function requestTimeout({ requestTimeout = 0 }) {
```

Comment:
@chulanovskyi-bs shouldn't this function be declared as async?

Reply:
it's not required, because we already return a Promise. And we don't need to use `await` inside the function

Reply:
Although your points are technically valid @chulanovskyi-bs, I personally think that specifying `async` in the signature of the function makes the contract more explicit. Just my 2 cents...

Reply:
🙂 ok